pkg/apis/serving/v1alpha1/revision_lifecycle.go

```diff
@@ -80,6 +81,15 @@ func (rs *RevisionSpec) GetContainer() *corev1.Container {
 	return &corev1.Container{}
 }
 
+// GetContainerConcurrency returns the container concurrency. If
+// container concurrency is not set, the default value will be returned
+func (rs *RevisionSpec) GetContainerConcurrency() int64 {
+	if rs.ContainerConcurrency == nil {
+		return config.DefaultContainerConcurrency
```

Comment:
@markusthoemmes @mattmoor mentioned to me on slack that the upgrade path old objects that had 0 will now have nil, so it might be good to have this fallback. WDYT?

Reply:
My thought is that this **actually** should fall back to the actually defined default value. I believe this currently falls back to the default-default value? I.e. the value that's chosen if no configmap setting is set?

Reply:
I have mixed feelings about which value to use.  I think the main use of this will be to properly handle the upgrade path from 0.8, in which case I think using `0` is more correct than the configured default.  I think I'd bias towards keeping what you have here, but adding a detailed comment explaining the upgrade-safe bias.